### DISCUSSION TITLE
Dist-Zilla Support

### DISCUSSION THREAD

**Discussion by rwp0** [Camelcade/Perl5-IDEA#2593] [Ideas]
Would be great to have a UI for `dzil new`, `dzil add` and friends since many (if not most) CPAN distributions are being build using this framework.

Are there any current plans, or future considerations regarding Dist-Zilla which has somewhat became the de-facto module builder framework for Perl?

Thanks.

**Comment by hurricup** [comment]
ye, there is a #1928 for start

Dist::Zilla is really nice, still have few issues:
- lot of dependencies 
- not working on old versions of perl

**Reply by rwp0** [reply]
I think the installing could be optional (as with `perlbrew` etc).

We can have a minimum Perl version required for this.

Supporting unsupported Perl versions (ie. the last two releases) is never recommended anyway.

Thanks for the consideration.
